app/views/admin/dashboard/dashboard.html.slim

```diff
@@ -40,10 +40,11 @@
       = content_tag(:div, t('common.no_records'), class: 'no-records-msg')
     - else
       - @people.each do |person|
-        h3 = person.name
-        .recent-projects
-          = render "recent_projects_grid_definition", grid: @people_grids["#{person}"]
-          = render_index_grid(@people_grids["#{person}"])
+        .person
+          h3 = person.name
+          .recent-projects
+            = render "recent_projects_grid_definition", grid: @people_grids["#{person}"]
+            = render_index_grid(@people_grids["#{person}"])
```

Comment:
I must have missed this before but `"#{person}"` is kind of an anti-pattern. Just `person` or maybe `person.to_s` is more appropriate. If this isn't clear let me know.